Yoast/wordpress-seo
# Reset indexables permalink after settings change

## Summary


This PR can be summarized in the following changelog entry:

* N/A Fixes a bug where the indexables' permalinks did not update after changing the settings.

## Test instructions

This PR can be tested by following these steps:

* Change permalink settings, verify in the database (Indexables table) whether the changes are updated. (See issue about steps how to reproduce the bug).
* Visit a post and verify that its permalink has been set (correct) again.

## Quality assurance


Fixes #13800
